.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Help us diagnose and fix bugs in Crossplane
+labels: bug
+---
+<!--
+Thank you for helping to improve Crossplane!
```

Comment:
Would it make sense to `s/Crossplane/Provider Template/g` in these files?

Reply:
I don't think so - we use "Crossplane" in the broader project sense for most provider repos that already have this issue template.